### DISCUSSION TITLE
badge finder

### DISCUSSION THREAD

**Discussion by PhyoYazar** [BEPb/BEPb#20] [Q&A]
Hi, glad to meet you and thank for your badge idea.

**Comment by Madkunt** [comment]
Hello, 
@BEPb
 I agree with @PhyoYazar the badge contribution idea you came up with is a great solution. 
Thanks 

> Hello everyone, I offer my help in obtaining this badge. I propose to add me to the followers and in my repositories ask 2 questions (create 2 discussions) to which I answer and you get this badge. After that, I create similar 2 discussions in your repositories and you answer them, so I will get the same badge. How do you like my idea? You can contact me by mail, the details are indicated on my page.

[https://github.com/orgs/github-community/discussions/18293?#discussioncomment-3163100]()
